websites/L/LTTStore.com/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+	clientId: "1258619078201839646",
+});
+
+const enum Assets {
+	Logo = "https://upload.wikimedia.org/wikipedia/commons/thumb/7/77/2018_Linus_Tech_Tips_logo.svg/512px-2018_Linus_Tech_Tips_logo.svg.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+	};
+	if (document.location.pathname === "/" || document.location.pathname === "") {
```

Comment:
The second condition is impossible. 
```suggestion
	if (document.location.pathname === "/") {
```